pyproject.toml

```diff
@@ -41,6 +41,7 @@ croniter = ">=1"
 defusedxml = "^0"
 pydantic = ">=1,<3"
 PyYAML = ">=5,<7"
+crcmod = "^1.7"
 
```

Comment:
It'd be nice if this would be an optional extra, given it's only used for this for now.

Reply:
done